### ISSUE TITLE
about the DINO training loss 

### ISSUE THREAD

**Issue by Faisal-Hajari** [facebookresearch/dino#225]
Hello, 
I'm training resnet18 on a custom dataset. it's been running for some time with a batch size of  325 (the max my gpu can handle). the thing is the loss is flat and it's not getting better or worse. 
is this behavior normal ? and if so how do you decide on stopping the training ?

**Comment by ComeBackCity** [comment]
Is it stuck around the 10-11 range?

**Comment by Cailei0412** [comment]
> Is it stuck around the 10-11 range?

Hi, did you solve the loss problem?
